library/src/G4CMPKaplanQP.cc

```diff
@@ -433,6 +444,20 @@ G4CMPKaplanQP::CalcQPAbsorption(G4double qpE,
   return EDep;
 }
 
+
+// Handle quasiparticle energy-dependent absorption efficiency
+
+G4double G4CMPKaplanQP::CalcQPEfficiency(G4double qpE) const {
+    G4double eff = absorberEff + absorberEffSlope * qpE;
```

Comment:
We'd like the slope to be a dimensionless quantity.  As written here, it has to have units of 1/energy.  Instead, I'd suggest we scale it by Delta (gapEnergy):

`G4double eff = absorberEff + absorberEffSlope * qpE/gapEnergy;`